src/main/java/org/opensearch/ad/model/MergeableList.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.ad.model;
+
+import java.util.List;
+
+public class MergeableList<T> implements Mergeable {
```

Comment:
Don't see this class used except in `MergeableListTests.java` , can we remove this class?

Reply:
The link doesn't show me the exact line, you can just write which line is using this class, I searched `MergeableList` on this PR page, seems just used in MergeableListTests.java,  maybe missed some folded code.

Reply:
Got it, looks just used it as general List, won't use the `merge` method. I guess the future change will use `MergeableList#merge` method. I'm ok with that.